src/sentry/integrations/vsts/issues.py

```diff
@@ -86,3 +88,45 @@ def get_issue(self, issue_id, **kwargs):
             'title': work_item['fields']['System.Title'],
             'description': work_item['fields'].get('System.Description')
         }
+
+    def sync_assignee_outbound(self, external_issue, user, assign=True, **kwargs):
+        client = self.get_client()
+        assignee = None
+
+        # TODO(LB): What's the scope here? is this correct?
+        # Get a list of all users in a given scope. How do we define scope?
+        # https://docs.microsoft.com/en-us/rest/api/vsts/graph/users/list?view=vsts-rest-4.1
+
+        if assign is True:
+            vsts_users = client.get_users(self.model.name)
+            sentry_emails = [email.email.lower() for email in user.get_verified_emails()]
+
+            for vsts_user in vsts_users['value']:
+                vsts_email = vsts_user.get(u'mailAddress')
+                if vsts_email and vsts_email.lower() in sentry_emails:
+                    assignee = vsts_user['mailAddress']
+                    break
+
+            if assignee is None:
```

Comment:
do you wanna throw a TODO in here to email people about failures like this? i think we'll want to eventually come back and do that for both this and jira